src/examples/java/io/nats/examples/NatsJsPullSubBase.java

```diff
@@ -13,9 +13,7 @@
 
 package io.nats.examples;
 
-import io.nats.client.JetStream;
-import io.nats.client.JetStreamSubscription;
-import io.nats.client.Message;
+import io.nats.client.*;
```

Comment:
I would usually not "refactor" for examples but I wanted to ensure that I used the same exact functionality for each example. If you really don't like this I can copy paste into each example.